cluster/deployment/scratchneta/.envrc.vars

```diff
@@ -39,3 +39,5 @@ export APPROVE_SV_RUNBOOK=true
 # TODO(DACH-NY/canton-network-node#14679): Remove
 export VALIDATOR_RUNBOOK_POSTGRES_PVC_SIZE="240Gi"
 export CACHE_DEV_DOCKER_REGISTRY=$DEV_DOCKER_REGISTRY
+
+export SELF_HOSTED_FLUENT_BIT=true
```

Comment:
This is actually deliberate for once because I did apply the cluster config here already.

My overall plan is to gradually apply it on all internal clusters, leave it on CILR for at least a week and then probably propagate to dev/test/mainnet with another week of delay each.